src/i18n.tsx

```diff
@@ -106,3 +106,9 @@ export function LanguageProvider({ children }: { children: ReactNode }) {
     </I18nProvider>
   )
 }
+
+// eagerly import the locale to unblock pageload
+;(() => {
+  const locale = initialLocale()
+  import(`@lingui/loader!./locales/${locale}.po`)
```

Comment:
couldn't this be done inside `LanguageProvider` so we can re-use hooks?

Reply:
This IIFE executes 65ms sooner than the LanguageProvider function component on a production build, so it must be here to be effective. It's definitely a niche optimization.